### ISSUE TITLE
HockeyApp SDK prevents exceptions from being show in XS/VSMac

### ISSUE THREAD

**Issue by VincentDondain** [bitstadium/HockeySDK-Xamarin#71]
See bug 51733 for reference: https://bugzilla.xamarin.com/show_bug.cgi?id=51733#c5

If you create a new iOS template, add hockey app SDK and add the following code in the AppDelegate's FinishedLaunching:

```
var manager = BITHockeyManager.SharedHockeyManager;
manager.Configure("37d9232914b841f387ec1f4ff35e9f52");
manager.StartManager();
manager.Authenticator.AuthenticateInstallation();

var storyboard = UIStoryboard.FromName ("Main", null);
var vc = storyboard.InstantiateViewController ("SomeViewControllerWithNoStoryboardID");
```

The exception will be captured by HockeyApp and not shown in the IDE.

Removing the `SharedHockeyManager` initialization code gets us back to the normal behavior, the exception is shown in the IDE.

**Comment by TroubleMakerBen** [comment]
Hey Vincent,
thank you for the pointer. We're looking into this =)

**Comment by TroubleMakerBen** [comment]
We have a fix in the pipeline that should make it to the release version soon. No ETA but it should happen within the next couple of days.

**Comment by TroubleMakerBen** [comment]
We've fixed this in 4.1.3 that was released just now (components haven't been approved yet, but the nuget is available already).

Let me know if you run into any issues, @VincentDondain, and thanks again for letting us know!

Benny

**Comment by TroubleMakerBen** [comment]
I needed to undo the fix for this as it introduced a regression that caused incomplete crash reports for .Net exceptions.
To fix the issue with the debugger for real, the native iOS SDK as well as the Xamarin SDK will need to undergo a significant refactoring which we can't do now as we have limited resources. It's on the backlog.
I'll update this issue accordingly once I know when we can fix it.
